libfwupd/fwupd-security-attr-private.h

```diff
@@ -265,13 +264,24 @@ G_BEGIN_DECLS
  **/
 #define FWUPD_SECURITY_ATTR_ID_INTEL_DCI_ENABLED "org.fwupd.hsi.IntelDci.Enabled"
 /**
- * FWUPD_SECURITY_ATTR_ID_INTEL_DCI_LOCKED:
+ * FWUPD_SECURITY_ATTR_ID_PLATFORM_FUSED:
  *
- * Host Security ID attribute for Intel DCI locked
+ * Host Security ID attribute for fused parts
  *
- * Since: 1.5.0
+ * Since: 1.8.0
+ **/
+#define FWUPD_SECURITY_ATTR_ID_PLATFORM_FUSED "org.fwupd.hsi.PlatformFused"
+/**
+ * FWUPD_SECURITY_ATTR_ID_PLATFORM_DEBUG_LOCKED:
+ *
+ * Host Security ID attribute for parts locked from debugging
+ *
+ * This was previously known as org.fwupd.hsi.IntelDci.Locked for Intel 1.5.0+
+ * It was renamed for all vendor support in 1.8.0.
```

Comment:
We've also got org.fwupd.hsi.IntelDci.Enabled as well as Locked; I'm really questioning whether we just want an org.fwupd.hsi.AmdPlatformDebug.Enabled -- I think it would be a lot more descriptive if we can link to per-vendor terms like DCI and JTAG.

Reply:
I think in this case the abstraction of details is better in documentation than in strings a user sees.

If I was an end user do I care if it's AMD Platform Debugging locked or Intel DCI locked? Probably not. I just care it's not locked down.

Reply:
I think it matters what is actually debug locked. I think it makes sense to rename:

 * org.fwupd.hsi.IntelDci.Enabled -> org.fwupd.hsi.CpuDebug.Enabled 
 * org.fwupd.hsi.IntelDci.Locked -> org.fwupd.hsi.CpuDebug.Locked

And then define new attributes:

 * org.fwupd.hsi.PlatformDebug.Enabled 
 * org.fwupd.hsi.PlatformDebug.Locked

I think trying to squash both into the same attribute is a bit of a pain; in essence they're the same, but it makes it really complicated to explain.

Reply:
I'm not convinced.  See the link from my other comment.

Reply:
Okay, I think we need to do org.fwupd.hsi.IntelDci.Enabled -> org.fwupd.hsi.Platform.Enabled still tho.